subsys/bluetooth/rpc/common/bt_rpc_gatt_common.c

```diff
@@ -16,7 +16,7 @@
 
 LOG_MODULE_DECLARE(BT_RPC, CONFIG_BT_RPC_LOG_LEVEL);
 
-#define ATTR_INDEX_POS 16
```

Comment:
Could you revert formatting changes so that it's clear what is actually changed?